### DISCUSSION TITLE
Dude this is sick.

### DISCUSSION THREAD

**Discussion by arcman7** [isaac-mason/recast-navigation-js#102] [General]
Nothing else to say, so far. Great work 💯

**Comment by isaac-mason** [comment]
I appreciate the kind words! If you give it a try, please let me know how you go 🙂

**Reply by arcman7** [reply]
Oh we will be using this in the near future; we're just getting our game engine setup at the moment. I briefly tested a few different npm packages for generating nav meshes, including yuka, but yours is incredibly performant. I now realize that you took the time to port a c++ library to a wasm executable and I really can't thank you enough for doing this. It may be a few months, but I'll show you what we build with this amazing tool :D
